Content.Server/GameObjects/Components/Atmos/GasAnalyzerComponent.cs

```diff
@@ -1,40 +1,61 @@
 ﻿#nullable enable
 using Content.Server.GameObjects.EntitySystems;
+using Content.Server.Interfaces;
 using Content.Shared.Atmos;
 using Content.Shared.GameObjects.EntitySystems;
+using Content.Shared.Interfaces;
+using Content.Shared.Interfaces.GameObjects.Components;
+using Content.Shared.Utility;
 using Robust.Shared.GameObjects;
 using Robust.Shared.GameObjects.Systems;
+using Robust.Shared.IoC;
+using Robust.Shared.Localization;
 using Robust.Shared.Map;
 using Robust.Shared.Utility;
 
 namespace Content.Server.GameObjects.Components.Atmos
 {
     [RegisterComponent]
-    public class GasAnalyzerComponent : Component, IExamine
+    public class GasAnalyzerComponent : Component, IAfterInteract
     {
+#pragma warning disable 649
+        [Dependency] private IServerNotifyManager _notifyManager = default!;
+        [Dependency] private ILocalizationManager _loc = default!;
```

Comment:
Don't need to pull this in.

Localisation manager is globally available as `Loc`, in `Robust.Shared.Localization`